src/main/java/de/wevsvirus/facades/ArztFacade.java

```diff
@@ -0,0 +1,45 @@
+package de.wevsvirus.facades;
+
+import com.github.javafaker.Faker;
+import de.wevsvirus.data.ArztRepository;
+import de.wevsvirus.model.ArztModel;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+
+import javax.annotation.Resource;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ArztFacade {
+
+    private Faker faker = new Faker();
+
+    @Resource
+    private ArztRepository arztRepository;
+
+    public List<ArztModel> findArztByPLZ(final String plz) {
+        String cleanPlz = StringUtils.trimAllWhitespace(plz);
+
+        List<ArztModel> aerzte = arztRepository.findByPlz(cleanPlz);
+        if (aerzte.size() > 0) {
+            return aerzte.stream()
+                    .limit(10L)
+                    .collect(Collectors.toList());
+        } else {
```

Comment:
Was tut dieser else zweig?

Reply:
Fake daten generieren falls keine dynamodb daten vorhanden sind

Reply:
Quasi für die lokale Entwicklung? Das würde ich raus nehmen in dieser Implementierung. Entweder man schaut, dass immer eine DB da ist z.b. mit ner H2, oder man macht ne alternative Implementierung z.b. mit Spring Profilen. Letzteres ist sehe einfach und du hast saubere Klassen. Kennst das?